package.json

```diff
@@ -15,8 +15,7 @@
     "tape": "~4.0.1"
   },
   "scripts": {
-    "test": "node test/index.js",
-    "prepublish": "npm ls && npm test"
```

Comment:
Not sure prepublish should be removed, definitely its removal shouldnt be mixed into this issue